site/content/docs/manifest/lb-web-service.md

```diff
@@ -39,6 +39,19 @@ List of all available properties for a `'Load Balanced Web Service'` manifest.
       LOG_LEVEL: info
     secrets:
       GITHUB_TOKEN: GITHUB_TOKEN
+
+    # Configuration for task storage using EFS volumes.
```

Comment:
Can we remove this comment and instead move this blurb to the backend service right below the `count` field?  

I think it should be more common for folks to enable EFS on a backend service rather than lb web service.